### PR TITLE
Lw/docs update 2d and speedtree

### PR DESCRIPTION
**Pull request by Katstodian**
Update missing 2D doc for LWRP.
Added SpeedTree docs.

### CODE REVIEW

File: com.unity.render-pipelines.lightweight/Documentation~/upgrading-your-shaders.md [hunk 1]
```diff
@@ -11,6 +11,8 @@ To upgrade built-in Shaders:
 
 **Tip:** If the Preview thumbnails in Project View are incorrect after you've upgraded, try right-clicking anywhere in the Project View window and selecting __Reimport All__.
 
+For [SpeedTree](https://docs.unity3d.com/Manual/SpeedTree.html) Shaders, Unity does not re-generate Materials when you re-import them, unless you click the Generate Materials or Apply & Generate Materials button.
```
Comment (phi-lira) [inline com.unity.render-pipelines.lightweight/Documentation~/upgrading-your-shaders.md]: Where are these buttons? Should we add an image or more detailed explanation here?

**Comment by Katstodian** [comment]
These empty spaces don't appear in my MD editing tool. Will fix here on GitHub.

File: com.unity.render-pipelines.lightweight/CHANGELOG.md [hunk 2]
```diff
@@ -12,12 +12,23 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Replaced beginCameraRendering callbacks by non obsolete implementation in Light2D
 - Updated `ScriptableRendererFeature` and `ScriptableRenderPass` API docs.
 
+### Added
+
+- Added documentation for SpeedTree Shaders in LWRP.
+
```
[the remark this answers is not in the document]
Comment (Katstodian) [reply com.unity.render-pipelines.lightweight/CHANGELOG.md]: Fixed

File: com.unity.render-pipelines.lightweight/CHANGELOG.md [hunk 3]
```diff
@@ -12,12 +12,23 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
...
 ### Fixed
+
 - Fixed a case where built-in Shader time values could be out of sync with actual time. [case 1142495](https://fogbugz.unity3d.com/f/cases/1142495/)
 - Fixed an issue that caused forward renderer resources to not load properly when you upgraded LWRP from an older version to 7.0.0. [case 1154925](https://issuetracker.unity3d.com/issues/lwrp-upgrading-lwrp-package-to-7-dot-0-0-breaks-forwardrenderdata-asset-in-resource-files)
 - Fixed GC spikes caused by LWRP allocating heap memory every frame.
 - Fixed NullReference exception caused when trying to add a ScriptableRendererFeature.
 
+- Added missing page for 2D Lights in LWRP.
+
```
[the remark this answers is not in the document]
Comment (Katstodian) [reply com.unity.render-pipelines.lightweight/CHANGELOG.md]: fixed

File: com.unity.render-pipelines.lightweight/Documentation~/upgrading-your-shaders.md (hunk 1 above)
Comment (Katstodian) [reply com.unity.render-pipelines.lightweight/Documentation~/upgrading-your-shaders.md]: We should - will update with next iteration of this doc. Please merge this :)

**Pull request merged**